components/match_ticker/commons/match_ticker.lua

```diff
@@ -217,6 +227,26 @@ function MatchTicker:buildQueryConditions()
 		conditions:add(teamConditions)
 	end
 
+	if Table.isNotEmpty(config.tiers) then
+		local tierConditions = ConditionTree(BooleanOperator.any)
+
+		Array.forEach(config.tiers, function(tier)
+			tierConditions:add { ConditionNode(ColumnName('liquipediatier'), Comparator.eq, tonumber(tier)) }
+		end)
+
+		conditions:add(tierConditions)
+	end
+
+	if Table.isNotEmpty(config.tierTypes) then
+		local tierTypeConditions = ConditionTree(BooleanOperator.any)
+
+		Array.forEach(config.tierTypes, function(tierType)
+			tierTypeConditions:add { ConditionNode(ColumnName('liquipediatiertype'), Comparator.eq, tierType) }
+		end)
+
+		conditions:add(tierTypeConditions)
+	end
+
```

Comment:
When none of the filters are selected, it marks the table as empty, which means this will return all matches while for the tournament list, it means that none of the tournaments are returned. Don't know if we expect the same behavior here

Reply:
I think it's fine for now. Let's get it out and deal with it then based on feedback. Personally think the matchlist version is better

Reply:
Fine for me as well